common/views/components/IIIFViewer/ViewerBottomBar.tsx

```diff
@@ -8,52 +7,7 @@ import { AppContext } from '@weco/common/views/components/AppContext/AppContext'
 import ItemViewerContext from '@weco/common/views/components/ItemViewerContext/ItemViewerContext';
 import useIsFullscreenEnabled from '@weco/common/hooks/useIsFullscreenEnabled';
 import ToolbarSegmentedControl from '../ToolbarSegmentedControl/ToolbarSegmentedControl';
-
-// TODO: update this with a more considered button from our system
-export const ShameButton = styled.button.attrs(() => ({
-  className: classNames({
-    'btn relative flex flex--v-center': true,
-    [font('hnb', 5)]: true,
-  }),
-}))<{ isDark?: boolean }>`
-  overflow: hidden;
-
-  ${props =>
-    props.isDark &&
-    `
-    border: 2px solid ${props.theme.color('transparent')};
-    color: ${props.theme.color('white')};
-    background: transparent;
-    outline: none;
-    transition: all ${props.theme.transitionProperties};
-
-    .btn__text {
-      position: absolute;
-      right: 100%;
-      @media (min-width: ${props.theme.sizes.large}px) {
-        position: static;
-      }
-    }
-
-    &:not([disabled]):hover {
-      border: 2px solid ${props.theme.color('white')};
-    }
-  `}
-
-  ${props =>
-    !props.isDark &&
-    `
-    background: ${props.theme.color('white')};
-    color: ${props.theme.color('green')};
-    border: 1px solid ${props.theme.color('green')};
-
-    &:not([disabled]):hover,
-    &:not([disabled]):focus {
-      background: ${props.theme.color('green')};
-      color: ${props.theme.color('white')};
-    }
-  `}
-`;
+import { ShameButton } from './ViewerTopBar';
```

Comment:
why is this called ShameButton again? does that imply that it needs attention, or is it just because it doesn't fit with the other Button components?

Reply:
It was something I picked up from Harry Roberts – basically we're openly embarrassed about this button not being part of our button system. It doesn't need attention from a functional perspective (https://csswizardry.com/2013/04/shame-css/)